playback/service/src/main/java/de/danoeh/antennapod/playback/service/internal/TimerValue.java

```diff
@@ -0,0 +1,19 @@
+package de.danoeh.antennapod.playback.service.internal;
+
+public class TimerValue {
+    private final long displayValue;
+    private final long milisValue;
```

Comment:
Wouldn't this be easier to understand if it had "type" and "value" instead of display/milis? I assume encoding the type in the value is the reason why you need two different ones?

Reply:
For time based sleep timers `displayValue` and `millisValue` are the same value, say 245245 millis left, but for episode sleep timers they're different, `displayValue` may be 1 (episode left), whereas `millisValue` will be the milliseconds left in the current episode.

Reply:
Okay, maybe a comment after displayValue could help.

```suggestion
    private final long displayValue; // Value shown to user (milliseconds or number of episodes)
    private final long milisValue;
```